runtime/doc/eval.txt

```diff
@@ -7008,6 +7009,28 @@ sqrt({expr})						*sqrt()*
 		"nan" may be different, it depends on system libraries.
 
 
+stdpath({what})					*stdpath()*
+		Returns the standard path(s) for {what}.
+
+		These directories are the default locations for various files
+		used by Neovim.
+
+		{what} can be one of the following:
+			'config'	User configuration directory. The
+					init.vim |vimrc| is stored here.
+					Returns a String.
+			'data'		User data directory. The |shada| is
```

Comment:
`shada` is a format. Format is not stored anywhere, need to refer to a `|shada-file|`.